tests/test_datastructures.py

```diff
@@ -70,6 +73,10 @@ def test_url_query_params() -> None:
     assert str(u) == "https://example.org/path/?order=name"
     u = u.remove_query_params("order")
     assert str(u) == "https://example.org/path/"
+    u = u.include_query_params(page=4, search="testing")
+    assert str(u) == "https://example.org/path/?page=4&search=testing"
+    u = u.remove_query_params(["page", "search"])
+    assert str(u) == "https://example.org/path/"
```

Comment:
What are you testing here?

Better to post the lines you are testing, otherwise, this will take some time for me.